### PR TITLE
Introduced the developer dashboard documentation

### PR DESCRIPTION
**Pull request by dimamost**

### CODE REVIEW

**Review by BraunMatthias — APPROVED** [review]

**Comment by BraunMatthias** [comment]
Also add note not to make use of real users from IdP (e.g. SAP users)

File: java/operating-applications/dashboard.md [hunk 1]
```diff
@@ -0,0 +1,198 @@
+---
+synopsis: >
+  This section describes how the CAP Developer Dashboard can be set up in both the local and cloud development environment to improve the developer experience.
+status: released
+redirect_from: java/observability
```
Comment (renejeglinsky) [inline java/operating-applications/dashboard.md:5]: Why is that redirect in here? Just a copy&paste leftover? ;)

File: java/operating-applications/dashboard.md [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+		}, 
+    ...
+	]
+}
+```
+:::
+
+Now you can deploy the application in BTP and assign the `cds.Developer` role to the users you want to grant access to the CAP Developer Dashboard.
+
+## Unauthorized Access
```
Comment (renejeglinsky) [inline java/operating-applications/dashboard.md:126]: That sounds to me as if sth has gone wrong or like a troubleshooting that explains what to do if someone did access it and was not authorized. How about "Disable Authorization"?

File: java/operating-applications/dashboard.md [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+</style>
+
+::: warning Only to be used in development
+The dashboard is only intended for use in the development environment. It is strictly forbidden to use the dashboard in a production environment, as it allows access to sensitive data and presents a security risk.
+:::
+<br>
+
+![Screenshot of the CAP developer dashboard UI.](assets/dashboard.jpg)
+
+
```
Comment (renejeglinsky) [inline java/operating-applications/dashboard.md:22]: This topic until now doesn't cover how to sue the dashboard. Would it make sense to link to the reCAP session on it?

https://broadcast.sap.com/replay/240604_recap?playhead=2188

File: java/operating-applications/dashboard.md [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+          roles:
+            - cds.Developer
+```
+:::
+
+## Cloud Setup
+
+If you also want to use the CAP Developer Dashboard in your cloud development scenario, you need to take a few more steps to achieve this. Let's take an example of a BTP Cloud Foundry app example with Approuter and XSUAA.
+
+First you must deactivate the [production profile](https://pages.github.tools.sap/cap/docs/java/developing-applications/configuring#production-profile). To do this, set the property `cds.environment.production.enabled` in `application.yaml` or `CDS_ENVIRONMENT_PRODUCTION_ENABLED` in `mta.yaml` to `false`.
```
Comment (renejeglinsky) [inline java/operating-applications/dashboard.md:57]: How about showing the config in codeblocks?

image

Comment (renejeglinsky) [reply java/operating-applications/dashboard.md]: @dimamost I prepared that locally. Should I commit it?

Comment (dimamost) [reply java/operating-applications/dashboard.md]: yes, sure

File: java/operating-applications/dashboard.md [hunk 5]
```diff
@@ -136,83 +136,57 @@ For security reasons, the **cds.Developer** role should only be used in conjunct
...
+In some cases, your application may run in a complex environment and you simply want to access the CAP Developer Dashboard running in your CAP Service Module directly without using a router in between. For this reason, you can switch off the authorization to grant direct unauthorized access.
 
-::: code-group
-```yaml [application.yaml]
-cds:
-  dashboard:
-    authorization:
-      enabled: false
-```
+1. Switch off authorization.
```
Comment (dimamost) [inline java/operating-applications/dashboard.md:141]: it is obvious that in this case either application.yaml or mta.yaml can be used

Comment (renejeglinsky) [reply java/operating-applications/dashboard.md:141]: Good point. It's not clear that it's _either/or_ and not _and_.

```suggestion
1. Switch off authorization either using one of the following options:
```

**Review by renejeglinsky — APPROVED** [review]

**Pull request merged**